src/platform/native.zig

```diff
@@ -305,6 +306,15 @@ pub const Platform = struct {
         return platform.last_window_size;
     }
 
+    pub fn hasEvent(platform: *Platform) !bool {
+        try glfw.pollEvents();
```

Comment:
I dont think hasEvents should poll, we already do that here: https://github.com/hexops/mach/blob/main/src/platform/native.zig#L488

Reply:
Thanks for making me rethink this.  You are right I can do without the poll.